tests/test_redirects.py

```diff
@@ -21,50 +21,22 @@ def test_that_checks_redirect_using_incorrect_query_values(self, base_url):
         }
         response = self._head_request(base_url, params=param)
 
-        Assert.equal(
-            response.status_code,
-            requests.codes.not_found,
-            self.response_info_failure_message(
-                base_url,
-                param,
-                response
-            )
-        )
+        assert (response.status_code == requests.codes.not_found,
+                self.response_info_failure_message(base_url, param, response))
```

Comment:
We've decided to be consistent with the order we have our expected vs actual when doing comparisons. Expected should be first. See our styleguide for details: https://wiki.mozilla.org/QA/Execution/Web_Testing/Docs/Automation/StyleGuide#Assertions